brew_wizard_ui/src/app/malt/malt.service.ts

```diff
@@ -14,27 +14,27 @@ export class MaltService {
   constructor(private http: HttpClient) { }
 
   getMalts(): Observable<IMalt[]> {
-    return this.http.get(`${environment.token_auth_config.apiBase}/malts`)
```

Comment:
This file is calls to the backend which have not been changed to fermentables, right?